app/lib/frontend/templates/views/layout.mustache

```diff
@@ -90,6 +90,7 @@
           <div class="account-nav-info"><span id="-account-profile-email">{{user_session.email}}</span></div>
           <div class="account-nav-divider"></div>
           <div class="account-nav-link"><a href="/my-packages">My packages</a></div>
```

Comment:
This reminds me that we'd need to update this template to use the urls from the `shared/urls.dart`, could you please add a `TODO` for it?

Reply:
Done